wayback-webapp/src/main/webapp/WEB-INF/replay/Toolbar.jsp

```diff
@@ -284,7 +287,8 @@ function trackMouseMove(event,element) {
        </td>
        <td style="padding:0!important;">
        <a style="position:relative; white-space:nowrap; width:<%= imgWidth %>px;height:<%= imgHeight %>px;" href="" id="wm-graph-anchor">
```

Comment:
When the `<a>` is outside of the `yearChart` (as it is in this PR) the `wm-graph-anchor` is being applied to `mCSB_1_scrollbar_horizontal` as well as `wm-ipp-sparkline`. 

![broader_anchor_markup](https://cloud.githubusercontent.com/assets/6740390/25356453/d61ae676-28ff-11e7-81f9-13a03baf2da9.jpg)

When the `<a>` is inside of the `yearChart` `<div>` it is only applied to `wm-ipp-sparkline`.

![narrower_anchor_markup](https://cloud.githubusercontent.com/assets/6740390/25356485/ed717a24-28ff-11e7-815e-54f14637bccc.jpg)

The dotted scrollbar ends up linked too--which I don't think is the desired behavior. The problem this is causing for me is that then when I scroll or click on the dotted scrollbar, it is linked and can take me to a different capture. If I have previously had the mouse pointer hovering over the year chart and the <a> href is now for a different date, I go to that date when I move the scrollbar any direction and then release.